api/src/main/java/org/pmiops/workbench/dataset/DataSetMapper.java

```diff
@@ -37,17 +42,47 @@
   @Named("dbModelToClientLight")
   DataSet dbModelToClientLight(DbDataset dbDataset);
 
+  @Mapping(target = "dataSetId", ignore = true)
+  @Mapping(target = "version", source = "etag")
+  @Mapping(target = "creatorId", ignore = true)
+  @Mapping(target = "creationTime", ignore = true)
+  @Mapping(target = "invalid", ignore = true)
+  @Mapping(target = "lastModifiedTime", ignore = true)
+  @Mapping(target = "prePackagedConceptSetEnum", ignore = true)
+  @Mapping(target = "values", source = "domainValuePairs")
+  DbDataset dataSetRequestToDb(DataSetRequest dataSetRequest);
```

Comment:
Can we add a test case for this in DataSetMapperTest?

Reply:
Done